web/packages/core/src/ruffle-player.ts

```diff
@@ -478,6 +482,13 @@ export class RufflePlayer extends HTMLElement {
         }
 
         this.unmuteAudioContext();
+        this.container.addEventListener("click", () =>
+            this.virtualKeyboard.blur()
+        );
+        this.virtualKeyboard.addEventListener(
+            "input",
```

Comment:
Isn't `"keydown"` more appropriate? Because then the pressed key is available directly from the event.

Reply:
There was a long discussion about this in the #frontend_apps channel on Discord. The issue is https://javascript.info/keyboard-events#mobile-keyboards.

Reply:
Tried keyup, keydown, and keypress. keyup and keydown have the same results: Nothing for e.code. "229" for e.keyCode and e.which. "Unidentified" for e.key. keypress does not fire.

Reply:
Also, this logic works when opening your Clipboard with the virtual keyboard and choosing a previously copied line of text. keydown would not work for that.

Reply:
![image](https://user-images.githubusercontent.com/29206584/221615348-8032cb4d-3b46-4b92-887d-080abe8fb07e.png)

Reply:
Got it.